HardcodeTray/modules/applications/b64electron.py

```diff
@@ -0,0 +1,88 @@
+"""
+Fixes Hardcoded tray icons in Linux.
+
+Author : Ivo Šmerek (ivo.smerek@gmail.com)
+Contributors : Bilal Elmoussaoui, Alexey Varfolomeev
+Website : https://github.com/bil-elmoussaoui/Hardcode-Tray
+Licence : The script is released under GPL, uses a modified script
+     form Chromium project released under BSD license
+This file is part of Hardcode-Tray.
+Hardcode-Tray is free software: you can redistribute it and/or
+modify it under the terms of the GNU General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+Hardcode-Tray is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with Hardcode-Tray. If not, see <http://www.gnu.org/licenses/>.
+"""
+from base64 import b64encode
+from glob import glob
+from os import path, system
+from fileinput import FileInput
+
+from HardcodeTray.app import App
+from HardcodeTray.modules.applications.electron import ElectronApplication
+from HardcodeTray.modules.applications.helpers.asar import AsarFile
+from HardcodeTray.modules.log import Logger
+from HardcodeTray.utils import get_pngbytes
```

Comment:
Codacy found an issue: [Cannot find implementation or library stub for module named 'HardcodeTray.utils' (error)](https://app.codacy.com/manual/RogueScholar/Hardcode-Tray/pullRequest?prid=5752280)